### PR TITLE
feat: FIP-0079: syscall for aggregated bls verification

### PR DESCRIPTION
**Pull request by hanabi1224**
This PR tries to implement https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0079.md without breaking FVM backward compatibility. 

(NOTE: This PR is largely based on https://github.com/filecoin-project/ref-fvm/pull/1914)

Closes https://github.com/filecoin-project/ref-fvm/issues/2001

Changes:
- Add `verify_bls_aggregate` syscall
- Add cargo feature `verify-signature` that keeps `verify_signature` syscall for backward compatibility
- Add integration test for the `verify-signature` feature
- Update CI to build and check the `verify-signature` feature

### CODE REVIEW

**Comment by jennijuju** [comment]
@DrPeterVanNostrand could you please help taking a look at this PR?

File: fvm/Cargo.toml [hunk 1]
```diff
@@ -58,3 +58,7 @@ m2-native = []
 upgrade-actor = []
 gas_calibration = []
 nv23-dev = []
+# Use this feature to remove `verify_signature` syscall that is supposed to be removed by FIP-0079,
+# The current implementation keeps it by default for backward compatibility reason.
+# See <https://github.com/filecoin-project/ref-fvm/issues/2001>
+no-verify-signature = []
```
Comment (Stebalien) [inline fvm/Cargo.toml:64]: Let's add this as a default feature. I.e.:

1. Add a `verify-signature` syscall feature.
2. Add that feature to the default feature's array.

Comment (hanabi1224) [reply fvm/Cargo.toml]: Fixed.

**Review by rvagg — APPROVED** [review]
lgtm, but grain-of-salt from me given my limited expertise on this

File: fvm/src/kernel/mod.rs [hunk 2]
```diff
@@ -220,6 +220,7 @@ pub trait ActorOps {
     fn balance_of(&self, actor_id: ActorID) -> Result<TokenAmount>;
 }
 
+#[cfg(feature = "verify-signature")]
```
[the remark this answers is not in the document]
Comment (Stebalien) [reply fvm/src/kernel/mod.rs:223]: It appears that the answer is no. Oh well...

File: fvm/src/gas/price_list.rs [hunk 3]
```diff
@@ -115,6 +115,11 @@ lazy_static! {
             }
         },
```
Comment (Stebalien) [inline fvm/src/gas/price_list.rs:116]: can we put `sig_cost` and `on_verify_signature` behind the feature-flag as well?

**Review by Stebalien — APPROVED** [review]
nit, otherwise lgtm!

File: fvm/src/gas/price_list.rs (hunk 3 above)
Comment (hanabi1224) [reply fvm/src/gas/price_list.rs:118]: Fixed.

**Review by Stebalien — APPROVED** [review]

**Pull request merged**